crates/pallets/starknet/src/lib.rs

```diff
@@ -271,6 +271,16 @@ pub mod pallet {
     #[pallet::getter(fn chain_id)]
     pub(super) type ChainId<T: Config> = StorageValue<_, Felt252Wrapper, ValueQuery>;
 
+    /// Current sequencer address.
+    #[pallet::storage]
+    #[pallet::getter(fn sequencer_address)]
+    pub type SequencerAddress<T: Config> = StorageValue<_, [u8; 32], ValueQuery>;
+
+    /// Ensure the sequencer address was updated for this block.
+    #[pallet::storage]
+    #[pallet::getter(fn seq_addr_update)]
+    pub type SeqAddrUpdate<T: Config> = StorageValue<_, bool, ValueQuery>;
+
```

Comment:
Is this necessary? Can't we use the `SequencerAddress` value itself to ensure it was updated for the block?

Reply:
I think it allows the SequencerAddress to stay in the storage, to be available for the off-chain worker, even after the `SeqAddrUpdate` is killed.

Reply:
We can't know if it has been updated or not by just having a value in `SequencerAddress`